### PR TITLE
Hotfix getpop

### PR DESCRIPTION
**Pull request by dcaseykc**

### CODE REVIEW

File: R/get_population.R [hunk 1]
```diff
@@ -335,7 +335,7 @@ get_population <- function(kingco = T,
                         sql_col = c('race_aian', 'race_as', 'race_blk', 'race_hisp', NA, 'race_nhpi', 'race_wht'))
   stopifnot(nrow(ref.table) == 7)
   ref.table = merge(ref.table, colnames, all.x = T, by = 'short')
-  if(!all(races == 'All')) races = ref.table[ short %in% races, value]
```
Comment (dcolombara) [inline R/get_population.R:338]: I'm not sure why this change would make a difference in addressing the problem identified by Carolina. I would have coded it the previous way. Regardless, it seems logical (though unconventional ... why have %in% single_string?

**Review by dcolombara — APPROVED** [review]
Good catch by Carolina. Nice simple fixes. In addition to my one comment, I'm curious about lines 153 & 155. Maybe you could enclose `unique(vals)` in a `return()` to make it more clear that it will be returned when the previous condition that returns 'All' is not true. Obviously, this isn't necessary, but might be nice for clarity for whoever inherits your code one day.

File: R/get_population.R (hunk 1 above)
Comment (dcaseykc) [reply R/get_population.R:338]: Yeah -- this change isn't really required. I just was trying to reduce instances of `==` for vectors with length >1
